buildmaster/LHCB_Z_13TEV_DIMUON_2022/metadata.yaml

```diff
@@ -0,0 +1,104 @@
+setname: "LHCB_Z_13TEV_DIMUON_2022"
+
+nnpdf_metadata:
+  nnpdf31_process: "Z0PT"
+  experiment: "LHCB"
+
+arXiv:
+  url: "https://arxiv.org/abs/2112.07458"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1990313"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1990313"
+  version: 2
+
+version: 1
+version_comment: "First implementation of the data set"
+
+implemented_observables:
+
+  # pT data
+  - observable: { description: "Differential cross-section of Z-->µµ as a function of Z-transverse momentum",
+                  label: '$\frac{d\sigma}{dp_T^Z}$',
+                  units: "[pb]"}
+    observable_name: PT
+    process_type: Z0PT
```

Comment:
This process type doesn't exist yet.

You will need to add a new process type for this dataset https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/process_options.py (at the bottom of the module you can find the ones currently implemented)

Reply:
This part is not really clear to me. In [LHCB_Z0_13TEV](https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/datafiles/new_commondata/LHCB_Z0_13TEV/metadata.yaml), `process_type` is set to `EWK_RAP` for the rapidity distribution. However, this does not appear at the bottom of the module that you mentioned. The same holds for [ATLAS_Z0J_8TEV](https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/datafiles/new_commondata/ATLAS_Z0J_8TEV/metadata.yaml), although here for the double distributions in pT. Am I missing something?

Reply:
Basically old datasets had the kinematic variables set to be always 3 variables and always the same one (for a given process, note that in this case "process" refer to both the physical process and the observable/measurement; so two datasets can have the same `nnpdf31_process`* and different `process_type`... it's confusing I know).

Anyway, the process_type basically defined how the variables are read by validphys and autogenerate some plotting labels. In the LHCB datasets this process_type still refers to the old process type because the kinematic variables are the same and written in the same order as before... 
I guess we can talk about this on Wednesday but basically the summary is that in order to add (new) DY data you will need to add a new process to that list that is able to understand the kinematic variables that you are using.